sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/README.md

```diff
@@ -88,7 +88,7 @@ spring.cloud.azure.appconfiguration.stores[0].monitoring.push-notification.secon
 
 Name | Description | Required | Default
 ---|---|---|---
-spring.cloud.azure.appconfiguration.stores[0].feature-flags.enable | Whether feature flags are loaded from the config store.  | No | false
+spring.cloud.azure.appconfiguration.stores[0].feature-flags.enabled | Whether feature flags are loaded from the config store.  | No | false
```

Comment:
This isn't related to the Evaluate flag enhancement is it?

Reply:
No, it was pointed out in a Github issue while I was working on it, so I figured I would add it in.